docs/src/api-testing-csharp.md

```diff
@@ -34,22 +34,19 @@ The following example demonstrates how to use Playwright to test issues creation
 GitHub API requires authorization, so we'll configure the token once for all tests. While at it, we'll also set the `baseURL` to simplify the tests.
 
 ```csharp
-using System;
```

Comment:
Q: Since we have now anyways the examples code for NUnit (ToT) and MSTest (this PR) we should maybe just introduce tabs here like we usually have?